src/calendar-heatmap.component.js

```diff
@@ -1783,13 +1770,13 @@ export class CalendarHeatmap extends Component {
     let minutes = Math.floor((seconds - hours * 3600) / 60);
     let time = '';
     if (hours > 0) {
-      time += hours === 1 ? '1 hour ' : hours + ' hours ';
+      time = hours === 1 ? `${time}1 hour ` : `${time}${hours} hours `;
     }
     if (minutes > 0) {
-      time += minutes === 1 ? '1 minute' : minutes + ' minutes';
+      time = minutes === 1 ? `${time}1 minute` : `${time}${minutes} minutes`;
```

Comment:
looks incorrect, `time +=` is changed to `time =`

Reply:
@maneetgoyal sir I have used the time variable before both of  the conditions' output as you can see. so `time += '1 minute' is similar to `${time}1 minute`

Reply:
study https://github.com/manufac-analytics/reactjs-calendar-heatmap/pull/23/commits/0cc654ac2eeaa87eb278b8e30bb7fe604542eb44